pom.xml

```diff
@@ -60,6 +60,7 @@ SPDX-License-Identifier: EPL-2.0
     <mdep.analyze.skip>true</mdep.analyze.skip>
     <!-- rat plugin is no good at validating EPL headers out of the box -->
     <rat.skip>true</rat.skip>
+    <revelc.min-build-mvn>3.9.0</revelc.min-build-mvn>
```

Comment:
Why not just require 3.9.8?  3.9.0 is pretty buggy, so was the first few.  3.9.8 is best stable.

Reply:
This is just the minimum. We can recommend newer. This minimum was derived automatically from the plugins configured in the pom.xml... they require at least 3.9.0.